build-artifact/action.yml

```diff
@@ -0,0 +1,6 @@
+name: 'docs-worker-actions'
+description: 'Build docs site html and save to artifact'
```

Comment:
minor, this does not build the docs site html - fetches the latest build data for given project